vite.config.ts

```diff
@@ -1,15 +1,14 @@
 import { defineConfig } from '@ownclouders/extension-sdk'
+import { id } from './public/manifest.json'
 
 export default defineConfig({
-  name: 'web-app-presentation-viewer',
-  server: {
-    port: 8082
-  },
+  base: '/vendor/apps/com.github.jankaritech.web.mdpresentation/',
```

Comment:
`/vendor/apps` from https://github.com/owncloud/ocis/blob/2606bcb555fc43b632683855ba95617ff941011e/services/web/pkg/service/v0/service.go#L32

Reply:
it also changed to `/assets/apps`